servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -481,82 +481,85 @@ void RendererSceneRenderRD::_render_buffers_post_process_and_tonemap(const Rende
 	}
 
 	{
-		RENDER_TIMESTAMP("Tonemap");
-		RD::get_singleton()->draw_command_begin_label("Tonemap");
+		RENDER_TIMESTAMP("PostProcess");
+		RD::get_singleton()->draw_command_begin_label("PostProcess");
```

Comment:
These changes should be reverted. The "Tonemap" pass only captures the very final stage of the post process pipeline. Renaming it to PostProcess will be misleading as most of the cost of post processing comes from the effects like DoF and Glow